ERROR:seismic_client(h1-m0):Error in SeismicClient.run() can't recover: (9, 'Bad file descriptor')
